### PR TITLE
[Scala] support str key type in kvstore

### PR DESCRIPTION
**Pull request by Ldpe2G**
follows #6765 
@javelinjs would you help to review ?

### CODE REVIEW

File: scala-package/core/src/main/scala/ml/dmlc/mxnet/LibInfo.scala [hunk 1]
```diff
@@ -117,16 +117,30 @@ private[mxnet] class LibInfo {
...
   @native def mxKVStorePush(handle: KVStoreHandle,
                             len: MXUint,
                             keys: Array[Int],
                             values: Array[NDArrayHandle],
                             priority: Int): Int
+  @native def mxKVStorePushEx(handle: KVStoreHandle,
+                            len: MXUint,
+                            keys: Array[String],
+                            values: Array[NDArrayHandle],
+                            priority: Int): Int
```
Comment (javelinjs) [inline scala-package/core/src/main/scala/ml/dmlc/mxnet/LibInfo.scala]: please align the argument indent. same for `mxKVStorePullEx`

**Comment by javelinjs** [comment]
LGTM. Could you help to check why jenkins test failed? I cannot open the link. @piiswrong

**Comment by piiswrong** [comment]
I can't open either. Something is wrong with this PR. Maybe close and open another

**Comment by Ldpe2G** [comment]
close and open another one

**Comment by Ldpe2G** [comment]
move to #6828

**Pull request closed without merging**